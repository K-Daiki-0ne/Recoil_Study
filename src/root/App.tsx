import React from 'react';
import { Home } from '../page/Home';
import { RecoilRoot } from 'recoil';

function App() {
  return (
    <RecoilRoot>
      <Home />
    </RecoilRoot>
  );
}

export default App;