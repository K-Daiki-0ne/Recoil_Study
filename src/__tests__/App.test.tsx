import React from 'react';
import App from '../root/App';
import Enzyme, { shallow } from 'enzyme';
import ReactSixteenAdapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new ReactSixteenAdapter() });


test('rendering test App.tsx', () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});