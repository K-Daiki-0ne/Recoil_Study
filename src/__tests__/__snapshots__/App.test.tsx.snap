// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render test rendering test App.tsx 1`] = `
<RecoilRoot>
  <Home />
</RecoilRoot>
`;

exports[`rendering test App.tsx 1`] = `
<RecoilRoot>
  <Home />
</RecoilRoot>
`;
